src/components/FeatureStyler/FeatureStyler.js

```diff
@@ -497,6 +591,30 @@ class FeatureStyler extends PureComponent {
       textRotation = featStyle.getText().getRotation();
     }
 
+    const oldStyles = FeatureStyler.getStyleAsArray(feature);
+
+    if (oldStyles.length > 1) {
+      for (let i = 1; i < oldStyles.length; i += 1) {
+        if (oldStyles[i].getImage() && oldStyles[i].getImage().getSrc()) {
+          for (let j = 0; j < lineIcons.length; j += 1) {
```

Comment:
Can you try to create a `findLineIcon(style, lineIcons)`   function ?

Reply:
Done. However, now we have more line changes.